### PR TITLE
Back button added

### PR DESCRIPTION
**Pull request by niteshkesarkar**

### CODE REVIEW

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html [hunk 1]
```diff
@@ -87,13 +87,22 @@
...
-                    <a routerLink="contribute/dashboard" *ngIf="this.router.url.includes('/contribute')" [class.active]="this.router.isActive('/contribute/dashboard', true)">{{ resourceService.frmelmnts.lnk.dashboard }}</a>
+                    <button
+                    *ngIf="!hideBackButton()"
+                    (click)="this.handleBack()"
+                    class="d-flex flex-ai-center sb-btn sb-btn-normal sb-btn-link sb-left-icon-btn sb-color-white" type="button" style="color:white;">
+                    <i class="icon-svg icon-svg--xs icon-arrow-long-left mr-8">
+                    <svg class="icon icon-svg--white">
+                        <use xlink:href="assets/images/sprite.svg#arrow-long-left"></use>
+                    </svg>
+                    </i>Back</button>
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/core/components/main-header/main-header.component.html]: Lang constant @niteshkesarkar

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html [hunk 2]
```diff
@@ -87,13 +87,22 @@
...
-                    <a routerLink="contribute/myenrollprograms" *ngIf="this.router.url.includes('/contribute')" [class.active]="this.router.isActive('/contribute/myenrollprograms', true)"><span class="sb-color-white">{{ resourceService.frmelmnts.lbl.myPrograms }}</span></a>
-                    <a routerLink="contribute/orglist" *ngIf="this.router.url.includes('/contribute')" [class.active]="this.router.isActive('/contribute/orglist', true)">{{ resourceService.frmelmnts.lbl.manageUsers }}</a>
-                    <a routerLink="contribute/dashboard" *ngIf="this.router.url.includes('/contribute')" [class.active]="this.router.isActive('/contribute/dashboard', true)">{{ resourceService.frmelmnts.lnk.dashboard }}</a>
+                    <button
+                    *ngIf="!hideBackButton()"
+                    (click)="this.handleBack()"
+                    class="d-flex flex-ai-center sb-btn sb-btn-normal sb-btn-link sb-left-icon-btn sb-color-white" type="button" style="color:white;">
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/core/components/main-header/main-header.component.html]: no inline style please @niteshkesarkar

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html (hunk 1 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/core/components/main-header/main-header.component.html]: Used resource bundle

File: src/app/client/src/app/modules/core/components/main-header/main-header.component.html (hunk 2 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/core/components/main-header/main-header.component.html]: removed extra inline css

**Pull request merged**